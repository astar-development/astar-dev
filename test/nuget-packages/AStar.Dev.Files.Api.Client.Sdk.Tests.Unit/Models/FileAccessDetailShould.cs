using AStar.Dev.Files.Api.Client.SDK.Models;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

public sealed class FileAccessDetailShould
{
    [Fact]
    public void ReturnTheExpectedToString() =>
        new FileAccessDetail
            {
                DetailsLastUpdated = new DateTime(2025, 12, 23) ,
                HardDeletePending  = true,
                Id                 = 1,
                LastViewed         = new DateTime(2025, 12, 24),
                MoveRequired       = true,
                SoftDeleted        = true,
                SoftDeletePending  = true
            }.ToString()
             .ShouldMatchApproved();
}
