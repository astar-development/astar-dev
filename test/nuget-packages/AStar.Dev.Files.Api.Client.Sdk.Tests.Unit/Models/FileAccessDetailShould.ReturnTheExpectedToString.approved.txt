{
  "id": 1,
  "detailsLastUpdated": "2025-12-23T00:00:00",
  "lastViewed": "2025-12-24T00:00:00",
  "softDeleted": true,
  "softDeletePending": true,
  "moveRequired": true,
  "hardDeletePending": true
}